app/views/admin/events/_form.html.erb

```diff
@@ -24,7 +24,5 @@
   <% end %>
   <%= f.submit "Submit", class: "btn btn-success" %>
   <%= link_to "Cancel", admin_events_path, class:"btn btn-default" %>
-  <% if @event.id %>
-    <%= link_to "Delete", admin_event_path(@event), class: "btn btn-default", method: :delete, data: { confirm: 'Are you sure?' }  %>
-  <% end %>
+  
```

Comment:
cool, changes so you can delete from the main listings and now you can just edit in the edit